src/types/types.js

```diff
@@ -265,6 +265,13 @@ export type EdgeMetadata = {
   notes?: string,
```

Comment:
This is missing recipientAddress param which will be used for all outgoing transaction including for swaps.

Reply:
This is handled a different way in the new design document.